nixos/modules/installer/tools/nixos-generate-config.pl

```diff
@@ -595,36 +638,178 @@ sub multiLineList {
   # services.openssh.enable = true;
 
   # Open ports in the firewall.
+
   # networking.firewall.allowedTCPPorts = [ ... ];
   # networking.firewall.allowedUDPPorts = [ ... ];
-  # Or disable the firewall altogether.
+
+  # WARNING! WARNING! WARNING! INSECURE!
+  #
+  # To disable the firewall altogether use the following.
+  #
+  # Use only inside trusted networks behind IPv4 NAT and without IPv6
+  # support or else bad people across the galaxy will be able to
```

Comment:
Only because the network has ipv6, does not mean your router does not block incoming connections to you IP.
And also NAT is not a firewall.

Suggestion:

```
Disable your firewall only in trusted networks with inbound firewalls or if your services are supposed to public on the internet.
```